ocean_lib/services/service.py

```diff
@@ -157,6 +167,19 @@ def as_dictionary(self) -> Dict[str, Any]:
         if self.description is not None:
             values["description"] = self.description
 
+        if self.additional_information is not None:
+            additional_information = {}
+            for key, value in self.additional_information.items():
```

Comment:
See my other (older) comment from test_service.py regarding this conversion. I think it is unnecessary.